lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -165,270 +169,261 @@ public Map<String, Object> handleRequest(JourneyRequest event, Context context)
                         new StringMapMessage()
                                 .with(
                                         LOG_MESSAGE_DESCRIPTION.getFieldName(),
                                         "F2F cri pending verification."));
-
-                return Optional.of(JOURNEY_PENDING);
+                return JOURNEY_PENDING;
             }
-            if (requestStatus.equals(CriResponseService.STATUS_ERROR)) {
+            case CriResponseService.STATUS_ERROR -> {
                 LOGGER.info(
                         new StringMapMessage()
                                 .with(LOG_MESSAGE_DESCRIPTION.getFieldName(), "F2F cri error"));
-                return Optional.of(JOURNEY_FAIL);
+                return JOURNEY_F2F_FAIL;
             }
+            default -> {
+                LOGGER.info(
+                        new StringMapMessage()
+                                .with(
+                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                        "F2F unexpected status: " + faceToFaceRequest.getStatus()));
+                return JOURNEY_F2F_FAIL;
+            }
+        }
+    }
+
+    private Map<String, Object> buildCiBreachingResponse(ContraIndicators contraIndicators)
+            throws CiRetrievalException, UnrecognisedCiException, ConfigException {
+        JourneyResponse journeyResponse = JOURNEY_FAIL_WITH_CI;
+
+        var cimitConfig = configService.getCimitConfig();
+        for (var ci : contraIndicators.getContraIndicatorsMap().values()) {
+            if (cimitConfig.containsKey(ci.getCode()) && !ci.isMitigated()) {
+                journeyResponse = new JourneyResponse(cimitConfig.get(ci.getCode()));
+                break;
+            }
+        }
+
+        StringMapMessage message = new StringMapMessage();
+        message.with(LOG_MESSAGE_DESCRIPTION.getFieldName(), "Returning CI error response.")
+                .with(LOG_ERROR_JOURNEY_RESPONSE.getFieldName(), journeyResponse.toString());
+        LOGGER.info(message);
+
+        return journeyResponse.toObjectMap();
+    }
+
+    private Map<String, Object> buildF2FNotCorrelatedResponse(AuditEventUser auditEventUser)
+            throws SqsException {
+        var message =
+                new StringMapMessage()
+                        .with(
+                                LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                "F2F return - failed to correlate VC data");
+        LOGGER.info(message);
+
+        auditService.sendAuditEvent(
+                new AuditEvent(
+                        AuditEventTypes.IPV_F2F_CORRELATION_FAIL,
+                        configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
+                        auditEventUser));
+
+        return JOURNEY_F2F_FAIL;
+    }
+
+    private Map<String, Object> buildNotCorrelatedResponse(AuditEventUser auditEventUser)
+            throws SqsException {
+        LOGGER.info(
```

Comment:
Could we be consistent with how we create the message for the logger, whether that's inline like here or separately like above

Reply:
Going to leave this for the review logging ticket :)